### PR TITLE
Update ruleset: Dell

### PR DESCRIPTION
**Pull request by injust**

### CODE REVIEW

**Comment by injust** [comment]
Fetch tests seem to be timing out on URLs that work just fine...

File: src/chrome/content/rules/Dell.xml [hunk 1]
```diff
@@ -87,7 +68,6 @@
 	<target host="dell.com" />
 	<target host="www.dell.com" />
 	<target host="accessories.dell.com" />
-	<target host="afcs.dell.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Dell.xml]: WFM

Comment (injust) [reply src/chrome/content/rules/Dell.xml]: A bunch of Google-indexed pages redirect to HTTP, e.g. http://afcs.dell.com/support/home/.

**Comment by J0WI** [comment]
This ruleset was whitelisted before, you could just update the hash and the exclusion bits there.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**